msb/rawlogger/msb_rawlogger.py

```diff
@@ -15,15 +15,16 @@ def signal_handler(sig, frame):
 
 
 class RawLoggerService:
-    def __init__(self, config: RawLoggerConf, subscriber: ZMQRawSubscriber):
+    def __init__(self, config: RawLoggerConf, subscriber: ZMQRawSubscriber, excluded_topics):
+        self.excluded_topics = excluded_topics
         self.config = config
         self.subscriber = subscriber
         self.logger = RawLogger(config)
-
+        print(self.logger,"self.logger")
```

Comment:
What is this print statement for?

Reply:
The print was old and for testing purposes. Print was removed